website/blog/2022-11-14-mongodb-crud-operations-with-ferretdb.md

```diff
@@ -130,34 +158,34 @@ This operation retrieves and displays all the documents present in the collectio
 Now, let's add a query parameter to the `find()` operation to filter for a specific item.
 
 ```js
-db.league.find({club:'PSG'})
+db.league.find({ club: "PSG" })
 ```
 
 This retrieves all the records that match the query:
 
-```js
+```sh
```

Comment:
Any reason why the format for mongosh output was changed from `js` to `sh`? I think `js` renders prettier in browser :thinking: 
js
![image](https://github.com/FerretDB/FerretDB/assets/24355139/1fbe6bbe-f2db-439d-b9e8-6a561f0e7727)

sh
![image](https://github.com/FerretDB/FerretDB/assets/24355139/2fd735e2-3843-492d-b446-bb605538bd5f)

Reply:
Because the output is not a valid JSON, and prettier makes it valid by adding the leading `;` in front of the output which looks strange :) After discussion with @Fashander we agreed to make output `sh`, so there are no strange symbols there.

Reply:
We also tried `json` format to have syntax highlight, but it modifies the output too (e.g. by adding `"` to the keys), so the output doesn't look equal to the results of `mongosh`

Reply:
Makes sense! Thanks for the explanation :hugs:

Reply:
> We also tried `json` format to have syntax highlight, but it modifies the output too (e.g. by adding `"` to the keys), so the output doesn't look equal to the results of `mongosh`

Yet the output is still modified and does not look equal to the results of `mongosh` (double quotes instead of single quotes, extra commas). So between "preserve output as is" and "use syntax highlighting", you chose "neither"? That's a strange choise

Reply:
because it was formatted before becoming sh